galleries/examples/statistics/boxplot_color.py

```diff
@@ -3,52 +3,34 @@
 Box plots with custom fill colors
 =================================
 
-This plot illustrates how to create two types of box plots
-(rectangular and notched), and how to fill them with custom
-colors by accessing the properties of the artists of the
-box plots. Additionally, the ``labels`` parameter is used to
-provide x-tick labels for each sample.
-
-A good general reference on boxplots and their history can be found
-here: http://vita.had.co.nz/papers/boxplots.pdf
+To color each box of a box plot individually:
+
+1) use the keyword argument ``patch_artist=True`` to create filled boxes.
+2) loop through the created boxes and adapt their color.
 """
 
 import matplotlib.pyplot as plt
 import numpy as np
 
-# Random test data
 np.random.seed(19680801)
-all_data = [np.random.normal(0, std, size=100) for std in range(1, 4)]
-labels = ['x1', 'x2', 'x3']
-
-fig, (ax1, ax2) = plt.subplots(nrows=1, ncols=2, figsize=(9, 4))
-
-# rectangular box plot
-bplot1 = ax1.boxplot(all_data,
-                     vert=True,  # vertical box alignment
-                     patch_artist=True,  # fill with color
-                     labels=labels)  # will be used to label x-ticks
-ax1.set_title('Rectangular box plot')
-
-# notch shape box plot
-bplot2 = ax2.boxplot(all_data,
-                     notch=True,  # notch shape
-                     vert=True,  # vertical box alignment
-                     patch_artist=True,  # fill with color
-                     labels=labels)  # will be used to label x-ticks
-ax2.set_title('Notched box plot')
+fruit_weights = [
+    np.random.normal(130, 10, size=100),
+    np.random.normal(125, 20, size=100),
+    np.random.normal(120, 30, size=100),
+]
+labels = ['peaches', 'oranges', 'tomatos']
+colors = ['peachpuff', 'orange', 'tomato']
```

Comment:
I did not know these colours existed by using them here is 😎

Reply:
Actually, I checked https://matplotlib.org/devdocs/gallery/color/named_colors.html and chose the fruits after available color names 😎